packages/object-lists/migrations/0000_bored_silver_surfer.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE `object_list_item` (
```

Comment:
Should we include the name `list` here? It bounds it to a specific context: an item in a list. If we, for some reason, decide to not use lists anymore but do want to continue with these items, we'd have to rename the table. For example: `object_item`?